exo-deploy/terraform/aws/alb/route53.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_route53_record" "external" {
```

Comment:
this was because there was a 1-1 mapping between a record and an ALB, while security groups were 1-many (1 security group would be reused for many ALBs). @Bowbaq - do you think it would be preferable to have each ALB create its own security group?